src/iccpd/src/mlacp_fsm.c

```diff
@@ -1291,3 +1586,90 @@ static void mlacp_exchange_handler(struct CSM* csm, struct Msg* msg)
 
     return;
 }
+
+/*****************************************
+ * Interface up ACK
+ *
+ ****************************************/
+static void mlacp_fsm_send_if_up_ack(
+    struct CSM       *csm,
```

Comment:
any possibility that csm may be NULL passed into this function?

Reply:
@gechiang , Thanks. ACK is sent as part of processing remote IF up event, and CSM can not be NULL at this point.